data/strings/strings.txt

```diff
@@ -4823,7 +4823,7 @@
 
   [data_version]
     comment = Data version in «About» screen, %@ is replaced by a local, human readable date.
-    tags = android,ios
+    tags = android
```

Comment:
@Jean-BaptisteC can Android's About screen be synced with iOS one, at least in used texts (and maybe also print app version at the title)? https://github.com/organicmaps/organicmaps/pull/7335